### ISSUE TITLE
bug: log-level=debug does not work anymore

### ISSUE THREAD

**Issue by alrevuelta** [waku-org/nwaku#2211]
When running this, I get debug logs.
```
docker run wakuorg/nwaku:v0.21.1 --log-level=DEBUG
```

When running this, no more debug logs are shown.
```
docker run wakuorg/nwaku:v0.21.3 --log-level=DEBUG
```

Seems to be a problem for v0.21.2 as well.

**Comment by chair28980** [comment]
@alrevuelta can you confirm this is still an issue in 0.21.3?

**Comment by alrevuelta** [comment]
Seems fixed in v0.22.0. With this DEBUG logs indeed appear.

```
docker run wakuorg/nwaku:v0.22.0 --log-level=DEBUG
```
